Tribler/Core/Modules/MetadataStore/store.py

```diff
@@ -204,7 +204,13 @@ def process_mdblob_file(self, filepath):
 
     @db_session
```

Comment:
I don't think we need to claim this `db_session`. `process_squashed_mdblob` already asks for the lock and `lz4.frame.decompress` doesn't need it. Could you include that in this PR?

Reply:
Yes, included the change.

Reply:
Thanks 👍